src/test/resources/features/15-attachment/attachments.feature

```diff
@@ -0,0 +1,19 @@
+ Feature: Attachments Management
+
+   Background:
+     Given a user logs in, searches a patient and visits Attachment page 
+     And a user clicks on Attachment link
+     Then the system loads Attachment page
+     And a user attaches a file
+     
+   Scenario: Uploading Attachment
+     And a user enters a caption
+     Then the system enables upload button
+     When a user clicks the upload link button 
```

Comment:
can it be both a link and button?

Reply:
I didn't take keen on this. However, have caught it in the next PR